sentry-core/src/main/java/io/sentry/core/SendCachedEvent.java

```diff
@@ -90,10 +103,18 @@ protected boolean isRelevantFileName(String fileName) {
 
   private void safeDelete(File file, String errorMessageSuffix) {
     try {
-      file.delete();
+      if (!file.delete()) {
+        logger.log(
+            SentryLevel.ERROR,
+            "Failed to delete '%s' " + errorMessageSuffix,
+            file.getAbsolutePath());
+      }
     } catch (Exception e) {
       logger.log(
-          SentryLevel.ERROR, "Failed to delete '%s' " + errorMessageSuffix, file.getName(), e);
+          SentryLevel.ERROR,
+          "Failed to delete '%s' " + errorMessageSuffix,
+          file.getAbsolutePath(),
```

Comment:
M: The format string is useful to avoid the allocation when logging is disabled so if we can get around passing `%s` instead of `+`, that'd be ideal:


```suggestion
          "Failed to delete '%s' %s",
          file.getAbsolutePath(),
          errorMessageSuffix,
```

Reply:
yep copy-pasta logs, it was like this already, fixed it